src/convertToObject.js

```diff
@@ -12,7 +12,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObj = {};
+  const convertStylesToArr = sourceString
+    .split('\n')
+    .filter(prop => prop.includes(':'))
+    .map(el => {
+      const style = el.slice(0, el.lastIndexOf(';')).split(':');
+
+      style[0] = style[0].trim();
+      style[1] = style[1].trim();
```

Comment:
use map after split(':') for trimming each value in resulting array of prop and it's value